### PR TITLE
fix for #1419

### PR DESCRIPTION
**Pull request by nesalang**
fixes issues for #1419 where when UIImage size is zero, app crashes.

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm [hunk 1]
```diff
@@ -404,7 +404,7 @@ - (void)processImageConcurrently:(std::shared_ptr<Image> const &)imageElem
                       // if view is available, set image to it, and continue image processing
                       if(view) {
                           view.image = img;
-                          if(imgElem->GetImageSize() == ImageSize::Auto || imgElem->GetImageSize() == ImageSize::Stretch || imgElem->GetImageSize() == ImageSize::None){
+                          if(img && (imgElem->GetImageSize() == ImageSize::Auto || imgElem->GetImageSize() == ImageSize::Stretch || imgElem->GetImageSize() == ImageSize::None)){
                               CGFloat heightToWidthRatio = img.size.height / img.size.width;
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: >img.size.width [](start = 77, length = 14)

Isn't it possible for `img` to be non-nil but have zero width? This code would still crash then.

Comment (bviglietta) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: (Crash or whatever NSLayoutConstraint does when given NaN or Inf.)

---
In reply to: [188021155](https://github.com/Microsoft/AdaptiveCards/pull/1444#discussion_r188021155) [](ancestors = 188021155)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required (VERSION 3.5)
-set(CMAKE_OSX_SYSROOT iphonesimulator11.2)
+set(CMAKE_OSX_SYSROOT iphonesimulator11.3)
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/CMakeLists.txt]: >3 [](start = 40, length = 1)

Is this change related to the PR?

Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/CMakeLists.txt]: yes, this updates the vesrion

---
In reply to: [188022357](https://github.com/Microsoft/AdaptiveCards/pull/1444#discussion_r188022357) [](ancestors = 188022357)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm (hunk 1 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: thanks

---
In reply to: [188022216](https://github.com/Microsoft/AdaptiveCards/pull/1444#discussion_r188022216) [](ancestors = 188022216,188021155)
